### PR TITLE
document D_802946E0

### PR DESCRIPTION
**Pull request by bates64**

### CODE REVIEW

File: src/battle/1A5830.c [hunk 1]
```diff
@@ -2702,6 +2700,15 @@ API_CALLABLE(RemoveActor) {
 }
 
 API_CALLABLE(DropStarPoints) {
+    /// Star Point multiplier, indexed by actor count.
+    /// +10% multiplier for two actors
```
Comment (HailSanta) [inline src/battle/1A5830.c:2704]: Isn't this 10% for 3, 30% for 4+?

Comment (bates64) [reply src/battle/1A5830.c:2704]: Don't know how I managed to mess this one up, lol

**Review by bates64 — COMMENTED** [review]
silly me

File: src/battle/1A5830.c [hunk 2]
```diff
@@ -2702,6 +2700,15 @@ API_CALLABLE(RemoveActor) {
 }
 
 API_CALLABLE(DropStarPoints) {
+    /// Star Point multiplier, indexed by actor count.
+    /// +10% multiplier for four actors
+    /// +30% multiplier for three or more actors
+    static s32 multiplier[] = {
```
Comment (ethteck) [inline src/battle/1A5830.c:2706]: I'd rather this not be static because it hides it from tools and other files. We could call it star point multiplier

**Pull request merged**